gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-all.zip
+# downgrade to 4.8.1 due to Lombok issues with 4.9
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.8.1-all.zip
```

Comment:
This cannot happen. Use of latest Gradle (currently 4.9) is **mandatory** as this is the version used to produce Docker images by Docker Hub. Downgrading Gradle would mean losing Docker.

Reply:
fixed the issue, no downgrade required